### PR TITLE
Add schema/docs for initial bulk import CSV format

### LINKED ISSUE #3: Define data format for bulk PII upload
Closely related to #2, define and document the data format for states to upload program participant records in bulk; e.g., [a CSV schema ](https://digital-preservation.github.io/csv-schema/). 

Defer any encryption container (covered in related issue #40)

### PR DESCRIPTION
**Pull request by ryanhofdotgov**
Closes #3

### CODE REVIEW

File: doc/bulk-import.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+cd piipan/doc/csv
+python3 validate.py example.csv
+```
+Any errors present in the supplied CSV file will be printed to `stdout`.
+
+## Notes
+- Microsoft Excel is notorious for mangling CSV files. By default, it will reformat date fields, remove leading zeros in integer-like strings, etc. Round-tripping CSV files using Excel is only possible through careful import and export:
+  1. Open a new workbook
+  1. Use File → Import, not File → Open 
+  1. When specifiying column format, hilight all the columns in the preview and select Text
```
Comment (kingcomma) [inline doc/bulk-import.md:23]:
```suggestion
  1. When specifying column format, highlight all the columns in the preview and select Text
```

**Review by kingcomma — COMMENTED** [review]
Added a couple suggestions. Let me know if my assumption about the first name field being required is incorrect.

File: doc/csv/import-schema.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        "description": "Participant's last name",
+        "constraints": {
+            "required": true,
+            "minLength": 1
+        }
+      },
+      {
+        "name": "first",
+        "type": "string",
+        "description": "Participant's first name"
```
[the remark this answers is not in the document]
Comment (ryanhofdotgov) [reply doc/csv/import-schema.json:15]: We will have to validate this detail with FNS (along with other items like diacritics and suffixes), but from my experience with the national background investigation system, it is acceptable for individuals just to have a one name.

Some trivia from [Wikipedia](https://en.wikipedia.org/wiki/Mononymous_person):
> The comedian and illusionist Teller, the silent half of the duo Penn & Teller, legally changed his original polynym, Raymond Joseph Teller, to the mononym "Teller" and possesses a United States passport issued in that single name

It's a big can of worms, but somewhere in between ["Falsehoods Programmers Believe About Names"](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/) and ASCII-only-everyone-has-a-first-and-last-name will be what this system will end up needing to implement. I'm guessing much, much closer to the latter.

[the remark this answers is not in the document]
Comment (kingcomma) [reply doc/csv/import-schema.json:15]: Yeah, in hindsight it's way too early to step into this rabbit hole. I was just thinking of the [SSA POMS](https://secure.ssa.gov/apps10/poms.nsf/lnx/0110212001) you had passed around that defines a legal name as consisting of both a first and last name. But I'm A-OK keeping it simple for now.

**Review by kingcomma — APPROVED** [review]
👍

**Pull request merged**